### PR TITLE
Introduce unary operators tests

### PR DESCRIPTION
**Pull request by lpawelcz**
Signed-off-by: Paweł Czarnecki <pczarnecki@antmicro.com>

### CODE REVIEW

File: .gitmodules
Comment (kamilrakoczy) [inline .gitmodules]: Please do not forget to remove this change after uhdm-integration merge.

Comment (lpawelcz) [reply .gitmodules]: It is now removed

**Review by kamilrakoczy — APPROVED** [review]
LGTM, lets wait for CI and I will merge this.

**Pull request merged**
